recipes/ansys-pythonnet/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ansys-pythonnet" %}
+{% set version = "3.1.0rc3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ansys-pythonnet-{{ version }}.tar.gz
+  sha256: 369a0a5a838a0991f755b6d63c319ab6997f9dc464d016187227be5cd860a9cb
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - mono  # [not win]
+    - dotnet ~=7.0
```

Comment:
Are these build dependencies? Should they be in host instead?

Reply:
Yes they are build dependencies.

Reply:
Note that host is "build" and build are compilers and other things. I know,  confusing, but b/c you don't seem to have a compiled extension, everything used to "build" should be in host.

Reply:
@SMoraisAnsys please check this. As far as my tests go you do not need these in the build section.

Reply:
@ocefpaf Thanks for the reminder, I didn't see your previous answer, I'll double check and clarify this in my head. Thanks for your time and care !